api.js

```diff
@@ -0,0 +1,141 @@
+
+const fetch = require('node-fetch');
+
+const express = require('express');
+
+const WebhookService = require('../services/webhook-service');
+const TranslationService = require('../services/translation-service');
+
+const { onshapeApiUrl } = require('../utils');
+
+const redisClient = require('../redis-client');
+    
+const apiRouter = express.Router();
```

Comment:
A general comment (that applies throughout the PR)

I don't think you need the initial blank line, I do think it's helpful to put a bit of vertical space between groups of related requires (especially when there's a lot of them), however beware having too much distance. In this case for instance the readability between the requires and the `express.Router()` is lost - it might be better to group lines 2/4 - then blank, then 6-7, then blank, then 9-11. Three groups with 2 blank sections between them.

Reply:
Updated in new push.